index.html

```diff
@@ -1,4 +1,7 @@
+<!DOCTYPE html><html lang=""><head><meta charSet="utf-8"/><meta http-equiv="X-UA-Compatible" content="IE=edge"/><title>Lightning · Framework for building TV apps</title>
+  <meta http-equiv="refresh" content="0; URL=https://lightningjs.io/">
+  <link rel="canonical" href="https://lightningjs.io/">
```

Comment:
@chiefcll wouldn't it be better to redirect to https://lightningjs.io/docs

Reply:
No - The current site is an overview of Lightning
https://rdkcentral.github.io/Lightning/

Reply:
Got it, I thought it was only the docs that were still there. But yeah, I guess redirecting to the lightningjs.io homepage works